Makefile

```diff
@@ -24,7 +24,7 @@ BUILD_DIR := $(TOPDIR)/build
 export TOPDIR
 
 ifeq ($(BUILD_GOPATH),)
-export BUILD_GOPATH=$(TOPDIR)/gopkgs
+export BUILD_GOPATH=$(TOPDIR)/build/gopkgs
```

Comment:
gopkgs was kept outside build directory to avoid downloading the dependencies every time.. The build directory gets deleted before every build. Downloading go packages every time increases compilation time. Why is this being moved inside build?

Reply:
Changed "make clean" to exclude build/gopkgs

Reply:
Agreed with Sachin.